src/mvTables.cpp

```diff
@@ -72,7 +72,9 @@ void mvTableColumn::handleSpecificKeywordArgs(PyObject* dict)
 	flagop("prefer_sort_descending", ImGuiTableColumnFlags_PreferSortDescending, _flags);
 	flagop("indent_enable", ImGuiTableColumnFlags_IndentEnable, _flags);
 	flagop("indent_disable", ImGuiTableColumnFlags_IndentDisable, _flags);
-
+	flagop("angle_header", ImGuiTableColumnFlags_AngledHeader, _flags);
+	flagop("disabled", ImGuiTableColumnFlags_Disabled, _flags);
```

Comment:
Instead of introducing new keywords (especially given the semantic conflict with the `enabled` argument), I'd suggest to finally fix the show/hide functionality on table columns.

1. `dpg.show_item()`, `dpg.hide_item()`, and the `show` argument should show/hide the column just like the built-in context menu does. This corresponds to `ImGui::TableSetColumnEnabled()`.
2. `dpg.is_item_shown()` should be synchronized with the `column->IsUserEnabled` flag (unfortunately there's no public API for that in ImGui). Alternatively, we can use `column->IsEnabled` but only on columns without `ImGuiTableColumnFlags_Disabled`, or even use `table->EnabledMaskByIndex`. Neither of these can be accessed via public API though.
3. `dpg.enable_item()`, `dpg.disable_item()`, and the `enabled` argument should control the `ImGuiTableColumnFlags_Disabled` flag.

I'm going to add a patch (via comments) that implements all the above, as soon as I make it work in my local repo.

Reply:
Here's the fix that implements enabled/show on columns:

https://github.com/v-ein/DearPyGui-fixes/compare/porting...feature/table-column-flags

BTW it turned out that (2) can be implemented via public API (`ImGui::TableGetColumnFlags`), which is good. No hacking.

Reply:
Thanks!!!